[2021-02-09 03:48:47,186] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_sample_dag.run_this 2021-02-09T00:46:35.257642+00:00 [queued]>
[2021-02-09 03:48:47,201] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_sample_dag.run_this 2021-02-09T00:46:35.257642+00:00 [queued]>
[2021-02-09 03:48:47,201] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2021-02-09 03:48:47,202] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2021-02-09 03:48:47,202] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2021-02-09 03:48:47,213] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): run_this> on 2021-02-09T00:46:35.257642+00:00
[2021-02-09 03:48:47,216] {standard_task_runner.py:53} INFO - Started process 7489 to run task
[2021-02-09 03:48:52,310] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_sample_dag.run_this 2021-02-09T00:46:35.257642+00:00 [running]> Maxims-MacBook-Air.local
[2021-02-09 03:48:57,339] {logging_mixin.py:112} INFO - hi
[2021-02-09 03:48:57,339] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-02-09 03:48:57,346] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=my_sample_dag, task_id=run_this, execution_date=20210209T004635, start_date=20210209T004847, end_date=20210209T004857
[2021-02-09 03:49:02,346] {logging_mixin.py:112} INFO - [2021-02-09 03:49:02,345] {local_task_job.py:103} INFO - Task exited with return code 0
